client/state/themes/utils.js

```diff
@@ -65,7 +65,13 @@ export function normalizeWpcomTheme( theme ) {
 		download_uri: 'download',
 	};
 
-	return mapKeys( theme, ( value, key ) => get( attributesMap, key, key ) );
+	if ( ! theme ) {
+		return {};
```

Comment:
Should this always return the same empty object instead of a new one?

Reply:
This is a fair question and what I did here is mimic the previous behaviour of `mapKeys` which always returns a new object (no matter `theme` is defined or not). So in this case we're early returning the output of
```js
Object.fromEntries( Object.entries( undefined ?? {} ) ).( /* ... */ )
```
 if that makes sense.

Reply:
Sure, that makes sense. 

This is just an extra optimization we could do at no cost as we're passing by. 

Leaving it to your final judgment anyway.